tests/test_benchmark.py

```diff
@@ -985,7 +969,7 @@ def lib_builder(name, native, env_init):
       return self.get_library('bullet', [os.path.join('src', '.libs', 'libBulletDynamics.a'),
                                          os.path.join('src', '.libs', 'libBulletCollision.a'),
                                          os.path.join('src', '.libs', 'libLinearMath.a')],
-                              configure_args=['--disable-demos', '--disable-dependency-tracking'], native=native, cache_name_extra=name, env_init=env_init)
+                              configure_args=['--disable-demos', '--disable-dependency-tracking', '--host=i686-unknown-linux'], native=native, cache_name_extra=name, env_init=env_init)
```

Comment:
why is this flag needed? and for which platform?

Reply:
This flag is needed for 2 reasons.
1) bullet in it's configure.ac tries to do platform detection and will fail on unknown platforms
2) More in general, configure will try to compile and run a test file to check if the C compiler is sane. As Cheerp will generate a wasm file which cannot be run, configure will fail. Passing ```--host``` enables cross compile mode, which lets configure complete happily.

Reply:
I see, thanks. Please add a comment with that (otherwise it looks very surprising! :)

Reply:
Comment added